build/ios/prepare_xcframework.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import glob
+import argparse
+import os
+import subprocess
+import shutil
+
+from pathlib import Path
+
+
+def main():
+    description = 'Create an xcframework from a framework & debug symbols'
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument('--framework-dir',
```

Comment:
consider using `_`-separated script parameters. This way it's much easier to just do Cmd+F and find where it's declared and where it's used.